examples/react/start-basic-react-query/app/routes/posts_.$postId.deep.tsx

```diff
@@ -1,11 +1,14 @@
 import { Link, createFileRoute } from '@tanstack/react-router'
 import { useSuspenseQuery } from '@tanstack/react-query'
-import { postQueryOptions } from '../utils/posts'
+import { convexQueryOptions } from 'convex-tanstack-query'
+import { api } from '../../convex/_generated/api'
 import { PostErrorComponent } from './posts.$postId'
 
 export const Route = createFileRoute('/posts/$postId/deep')({
   loader: async ({ params: { postId }, context }) => {
-    const data = await context.queryClient.fetchQuery(postQueryOptions(postId))
+    const data = await context.queryClient.fetchQuery(
```

Comment:
This isn't your error, but let's change these `fetchQuery` calls to `ensureQueryData`. Thoughts?

Reply:
made this change in https://github.com/TanStack/router/pull/1962